context/ometiff_tiler.py

```diff
@@ -18,29 +28,100 @@ def tile_ometiff(filename, channel_pages, output_directory, prefix):
 
         pyvips.Image.dzsave(image, os.path.join(path, channel), overlap=0, tile_size=256)
 
+def tile_tiff(filename, output_directory, prefix, server_url):
+    tile_size = 512
+    image = pyvips.Image.tiffload(filename)
+    metadata_str = image.get('image-description')
+    namespaces = {'http://www.openmicroscopy.org/Schemas/OME/2016-06': None,
+                          'http://www.openmicroscopy.org/Schemas/SA/2016-06': None}
+    output_metadata = {}
+    image_metadata = xmltodict.parse(
+        metadata_str,
+        process_namespaces=True,
+        namespaces=namespaces
+    )['OME']['Image']['Pixels']
+    channel_metadata = image_metadata['Channel']
+    output_metadata['width'] = int(image_metadata['@SizeX'])
+    output_metadata['height'] = int(image_metadata['@SizeY'])
+    output_metadata['tile_size'] = tile_size
+    output_metadata['channels'] = {}
+    output_subdirectory = f'{prefix}.images'
+    path = os.path.join(
+        output_directory,
+        output_subdirectory
+    )
+    print(f'Writing images to {path}')
+    for channel in channel_metadata:
+        channel_info = channel['@ID'].split(':')
+        # Channel1:0
+        if(re.match('Channel([0-9]+)', channel_info[0])):
+            output_metadata['channels'][channel['@Name']] = int(channel_info[0][-1])
+        # Channel:0:1
+        else:
+            output_metadata['channels'][channel['@Name']] = int(channel_info[-1])
+    for channel in output_metadata['channels']:
+        if not os.path.exists(path):
+            os.mkdir(path)
```

Comment:
[This](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir) would mimic `mkdir -p`.